exporter/awsxrayexporter/internal/translator/segment.go

```diff
@@ -172,8 +174,8 @@ func MakeSegment(span ptrace.Span, resource pcommon.Resource, indexedAttrs []str
 		}
 	}
 
-	if name == "" && span.Kind() == ptrace.SpanKindServer {
-		// Only for a server span, we can use the resource.
+	if name == "" && (span.Kind() == ptrace.SpanKindServer || span.Kind() == ptrace.SpanKindConsumer) {
+		// Only for a server and consumer spans, we can use the resource.
 		if service, ok := resource.Attributes().Get(conventions.AttributeServiceName); ok {
```

Comment:
I want to double check if it makes sense to have consumer span (segment) name as `service.name`? If consumer span is a span with parent, then the SQS node will be merged into service node as one node.

Reply:
Yes, this is the desired behavior.